source/_docs/guides/drupal-8-composer-no-ci.md

```diff
@@ -0,0 +1,236 @@
+                "DrupalProject\\composer\\ScriptHandler::createRequiredFiles"
+            ],
+            "post-create-project-cmd": [
+                "@drupal-scaffold",
+                "DrupalProject\\composer\\ScriptHandler::createRequiredFiles"
+            ]
+        },
+    ```
+  </div>
+</div>
+
+## Managing Drupal with Composer
+
+### Downloading Drupal Dependencies with Composer
+
+Normally the next step would be going through the standard Drupal installation. But since we’re using Composer, none of the core files exist yet. Let’s use Composer to install Drupal core.
+
+Since we modified `composer.json` we will need to update Composer to download the new dependencies. 
+
+```
+composer update
+```
+
+This may take a while as all of Drupal core and its dependencies will be downloaded. Subsequent updates should take less time.
+
+![image of terminal running a composer install](/source/docs/assets/images/guides/drupal-8-composer-no-ci/drops-8-composer-install.png)
+
+```
+git status
+```
+
+**Important:** The `example-drops-8-composer` `.gitignore` file assumes that you’re using a build step with continuous integration. To make it compatible with this method we need to remove everything above the CUT section of the `.gitignore` file. Without this modification critical components, such as Drupal core and contrib modules, will be ignored and not pushed to Pantheon. Now let’s run git status again to make sure everything is included.
+
+```
+git status
+```
+
+![image of git status showing the changed files in red](/source/docs/assets/images/guides/drupal-8-composer-no-ci/drops-8-composer-git-status-after-installing-d8.png)
+
+Set the site to git mode.
+
+```
+terminus connection:set $PANTHEON_SITE_NAME.dev git
+```
+
+Add and commit the code files. A git force push is necessary initially but subsequent pushes should not need `--force`.
+
+```
+git add .
+
+git commit -m 'Drupal 8 and dependencies'
+
+git push --force
+```
+
+**Note** the `vendor` directory is being committed to Pantheon. This is because Pantheon needs the full site artifact. If you prefer to ignore the `vendor` directory then take a look at our documentation on the more advanced automated workflow with a build step.
+
+### Installing Drupal
+
+Now that the code for Drupal core exists on our Pantheon site, we need to actually install Drupal.
+
+Change to SFTP mode as files need to be writeable on Pantheon in order to install Drupal.
+
+```
+terminus connection:set $PANTHEON_SITE_NAME.dev sftp
+```
+
+Use Terminus Drush to install Drupal.
+
+```
+terminus drush $PANTHEON_SITE_NAME.dev -- site-install -y
+```
+
+Log in to your new Drupal 8 site to verify it is working. You can get a one-time login link using Drush.
+
+```
+terminus drush $PANTHEON_SITE_NAME.dev -- uli
+```
+
+**Note:** there is no need to commit changes as the `example-drops-8-composer` project that we started with has already defined `$settings['install_profile'] = 'standard';` in `settings.php`.
```

Comment:
Do we need to the switch back and forth at all? I was just able to do a `drush si` on a site in git mode that had the `settings.php` piece in place.